src/lando/api/tests/test_landings.py

```diff
@@ -553,6 +555,78 @@ def test_merge_conflict(
             assert re.match(f"{repo.pull_path}/file", fp["url"])
 
 
+@pytest.mark.parametrize(
+    "repo_type,bad_commit_type",
+    # We make a cross-product of all the SCM and all the bad actions.
+    # As we don't want a cross-product of bad actions and reasons, we bundle them in a
+    # tuple, that we deconstruct in the test.
+    itertools.product(
+        [SCM_TYPE_HG, SCM_TYPE_GIT],
+        # All of BAD_COMMIT_TYPES, but not wpt
```

Comment:
I think this and maybe other instances of "bad" may be outdated based on the target branch.